experimental/grouped-and-auto-accessors.md

```diff
@@ -0,0 +1,85 @@
+# [Grouped Accessors and Auto-Accessors][proposal-grouped-and-auto-accessors]
+
+## ClassBody
+
+```js
+extend interface ClassBody {
+    body: [ MethodDefinition | PropertyDefinition | StaticBlock | ClassAccessor ];
+}
+```
+
+## ObjectExpression
+
+```js
+extend interface ObjectExpression {
+    properties: [ Property | ObjectAccessor ];
+}
+```
+
+## ClassAccessor
+
+```js
+interface ClassAccessor <: PropertyDefinition {
+    type: "ClassAccessor";
+    get: GetAccessorStub | GetAccessorMethod | null;
+    set: SetAccessorStub | SetAccessorMethod | null;
+}
+```
+
+If `set.private` is `true`, `key` must be an `Identifier` and `computed` must be `false`.
+
+## ObjectAccessor
+
+```js
+interface ObjectAccessor <: Property {
+    type: "ObjectAccessor";
+    get: GetAccessorStub | GetAccessorMethod | null;
+    set: SetAccessorStub | SetAccessorMethod | null;
+}
+```
+
+### GetAccessorStub
```

Comment:
Why use “stub” instead of using the proposal language, such as GetAutoAccessor?

Reply:
The "stub" comes from the [proposed syntax section](https://github.com/tc39/proposal-grouped-and-auto-accessors#proposed-syntax).

According to the spec, an auto-accessor is a simplified version of grouped accessor, e.g. `accessor x = { get; set; }`, where the accessor stub refers to the `set` and `get` shorthand.

Reply:
Ah ok.